src/renderer/components/select/select.test.tsx

```diff
@@ -37,6 +38,10 @@ describe("<Select />", () => {
 
     di.override(directoryForUserDataInjectable, () => "some-directory-for-user-data");
     di.override(rendererExtensionsInjectable, () => computed(() => [] as LensRendererExtension[]));
+    di.override(ipcRendererInjectable, () => ({
+      on: jest.fn(),
```

Comment:
I think it would be better to override mocks by the specific unit tests, instead of generally here. The requirements for the mocks vary a lot, and trying to use shared to serve all purposes is an example of a wrong abstraction.

Also the wrong thing is mocked here: `ipcRendererInjectable` is a volatile, untrusted module from a foreign source (Electron). It is better to override the clean, minimal and local abstractions leading to it.

Reply:
I know but the better thing to do would be to actually replace all of this with proper IPC abstraction. So I don't think it is worth it here.

Reply:
True, resolved.